yarnrunner_python/runner.py

```diff
@@ -147,8 +154,27 @@ def choose(self, index):
         self._vm_data_stack.insert(0, choice["choice"])
         self.resume()
 
-    def add_command_handler(self, command, fn):
-        self._command_handlers[command] = fn
+    def add_command_handler(self, command: str, cmd: Callable):
+        """
+        Registers a custom command that can be invoked from Yarn scripts.
+        Like `<<get_player_name>>` or `<<walk MyCharacter StageLeft>>`.
+        Usually these commands don't return anything. Returning a string, will put it on the line where is used.
```

Comment:
> Returning a string, will put it on the line where is used.

Is this how Yarn handles this in Unity? I was under the impression that the return value had no bearing on the story and that this was a method to trigger events within the external "game".

Reply:
Ah, that's fine, I didn't know that was the current runner behavior. Thanks for catching that!